src/static/css/2019.css

```diff
@@ -1223,3 +1223,97 @@ p.copyright a {
     margin-top: 20px;
   }
 }
+
+.nav-dropdown {
+  position: relative;
+  margin-right: 2vw;
+}
+
+.nav-dropdown-btn {
+  padding: 20px 45px 20px 20px;
+  font-size: 17px;
+  font-size: 1.0625rem;
+  text-decoration: none;
+  position: relative;
+  display: flex;
+  align-items: center;
+  border: none;
+  border-radius: 25px;
+  background: #44536a;
+  color: white;
+}
+
+.nav-dropdown-btn.dropdown-open {
+  border-radius: 25px 25px 0 0;
+}
+
+.nav-dropdown-btn::after {
+  content: "";
+  position: absolute;
+  pointer-events: none;
+  border-style: solid;
+  border-width: 2px 2px 0 0;
+  border-color: currentColor;
+  height: 7px;
+  width: 7px;
+  top: 50%;
+  margin-top: -7px;
+  right: 23px;
+  -webkit-transform: rotate(135deg);
+  -ms-transform: rotate(135deg);
+  transform: rotate(135deg);
+  transition: all 200ms linear;
+}
+
+.nav-dropdown-btn.dropdown-open::after {
+  width: 17px;
+  right: 16px;
+  margin-top: -9px;
+}
+
+.nav-dropdown-list {
+  align-items: flex-start;
+  flex-direction: column;
+  user-select: none;
+  font-size: 17px;
+  font-size: 1.0625rem;
+  color: #fff;
+  position: absolute;
+  z-index: 1;
+}
+
+.nav-dropdown-list.align-right {
+  right: 0;
+}
+
+.nav-dropdown-list li {
+  width: 100%;
+}
+
+.nav-dropdown-list a {
+  background: #44536a;
```

Comment:
Reusing this color from the footer background
```suggestion
  background: #5c687d;
```

Reply:
This change looks fine in header:

https://drive.google.com/file/d/1vLmcl3WhgrK0Qd_5XVK8D56CQCjoNlZo/view

but in footer background of list matches with footer so it will look like this

![Screenshot 2020-08-16 at 9 36 13 PM](https://user-images.githubusercontent.com/10291571/90338610-84c3b500-e008-11ea-86de-a19ed441ca44.png)

BTW in the video above check if other changes you suggested are as expected, transparent bg for button, and white border, and color changes to (#f7f779) yellow on hover and focus

Reply:
I don't think it even looks fine in header btw. Not enough contrast their either IMHO so think we need the dark background.

Reply:
Per https://github.com/HTTPArchive/almanac.httparchive.org/pull/1214#discussion_r471014005 I'd also suggest using a border color on the dropdown, which should help distinguish it.

Reply:
Changed, footer now open upwards